### DISCUSSION TITLE
Expo Go Beta SDK 50 Android?

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [expo/expo#26383] [Archived: Questions]
While i found the Testflight for iOS I havent found a link for Android?

**Comment by drmolixcool** [comment]
here the link to expo go compatible with sdk 50 : https://d1ahtucjixef4r.cloudfront.net/Exponent-2.30.8.apk

Source : the endpoint url is same that stable version of Expo Go released on official website https://expo.dev/tools in download section : https://d1ahtucjixef4r.cloudfront.net/Exponent-2.29.8.apk

**Reply by jhay06** [reply]
is Expo Go apk opensource ?

**Reply by MrWiki15** [reply]
no
